downstream/assemblies/platform/assembly-controller-inventory-templates.adoc

```diff
@@ -1,4 +1,4 @@
-[id="assembly-inventory-templates"]
+[id="controller-inventory-templates"]
```

Comment:
I'd still have the assembly- in there.

Reply:
I removed it because when it builds on the portal, "assembly" appears in the customer facing link.